docker-compose.yml

```diff
@@ -25,8 +25,8 @@ services:
 
 
   snowstorm:
-    image: snomedinternational/snowstorm:latest
-    container_name: snowstorm
+    image: snomedinternational/snowstorm-x:latest
+    container_name: snowstorm-x
```

Comment:
I will change this back to snowstorm after merge as we need to update the ES version too.